RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1,5 +1,28 @@
 //A file full of utility functions which we import into autoscend.ash
 
+boolean almostRollover()
+{
+	int warning_time = get_property("auto_stopMinutesToRollover").to_int() * 60;
+	int remaining_time = rollover() - (now_to_int()/1000);
+	auto_log_debug(`Less than {(remaining_time/60)+1} min until rollover, bedtime at {warning_time/60} min`, "blue");
```

Comment:
almostRollover is called every loop in doTasks, it's going to spam a lot of someone enables debug logging. Maybe change to only print if we're like 5 minutes away from the stop time.

Reply:
Thanks for the advice.  Done!